homeassistant/components/homeassistant/triggers/event.py

```diff
@@ -25,12 +25,23 @@
 )
 
 
-def _populate_schema(config, config_parameter):
-    if config_parameter not in config:
+def _schema_value(value, exact_match):
+    if isinstance(value, list) and not exact_match:
+        return vol.In(value)
+
+    return value
+
+
+def _populate_schema(config, config_parameter, exact_match=True):
```

Comment:
This is really messy. It's no longer possible to reuse code. Remove both `_schema_value` and `_populate_schema` and do it inline.